src/astro/sql/__init__.py

```diff
@@ -40,16 +47,19 @@ def run_raw_sql(
     database: Optional[str] = None,
     schema: Optional[str] = None,
     handler: Optional[Callable] = None,
-):
-    return transform_decorator(
+    **kwargs,
+) -> TaskDecorator:
+
+    kwargs["conn_id"] = conn_id
+    kwargs["parameters"] = parameters
+    kwargs["database"] = database
+    kwargs["schema"] = schema
+    kwargs["handler"] = handler
```

Comment:
wdyt about the following?
```suggestion
    kwargs.update({
        "conn_id": conn_id,
        "parameters": parameters,
        "database": database,
        "schema": schema,
        "handler": handler,
    })
```